PROVISIONING-METAL.md

```diff
@@ -265,6 +265,35 @@ Older networking configuration versions (such as `1` or `2`) are supported in ne
 | Version 2                     | [v1.10.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.10.0) |
 | Version 3                     | [v1.12.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.12.0) |
 
+#### Validate network configuration
+
+`netdog` has a command `validate-net-config` to validate that network configuration files parse correctly.
+This command is intended to validate the format and structure of the file, it will not guarantee the generated configuration will work in a particular context.
+The command can be run by passing the path to a `net.toml` file:
+```bash
```

Comment:
nit: throughout this file we are separating code blocks from text with a newline.

```suggestion
The command can be run by passing the path to a `net.toml` file:

```bash
```